apps/export/tests/test_apis.py

```diff
@@ -129,6 +132,33 @@ def test_export_filter_by_status(self):
         response = self.client.get(f'/api/v1/exports/?status={Export.PENDING},{Export.FAILURE}')
         assert response.json()['count'] == 2
 
+    def test_export_filter_by_type(self):
+        self.create(Export, exported_by=self.user, type=Export.ENTRIES)
+        self.create(Export, exported_by=self.user, type=Export.ASSESSMENTS)
+        self.create(Export, exported_by=self.user, type=Export.ASSESSMENTS)
+        self.create(Export, exported_by=self.user, type=Export.PLANNED_ASSESSMENTS)
+
+        self.authenticate()
+        response = self.client.get(f'/api/v1/exports/?type={Export.ASSESSMENTS}')
+        assert response.json()['count'] == 2
+        response = self.client.get(f'/api/v1/exports/?type={Export.ASSESSMENTS},{Export.ENTRIES}')
+        assert response.json()['count'] == 3
+
+    def test_export_filter_by_exported_at(self):
+        now = timezone.now()
+        self.update_obj(self.create(Export, exported_by=self.user), exported_at=now + relativedelta(days=+2))
+        self.update_obj(self.create(Export, exported_by=self.user), exported_at=now + relativedelta(days=+3))
+        self.update_obj(self.create(Export, exported_by=self.user), exported_at=now + relativedelta(days=+4))
+        self.update_obj(self.create(Export, exported_by=self.user), exported_at=now + relativedelta(days=-2))
```

Comment:
This will be more cleaner @k9845 , what do you think 
```
days = [2,3,4,-2]
for d in days:
    self.update_obj(self.create(Export, exported_by=self.user), exported_at=now + relativedelta(days=d))
```